package.json

```
@@ -13,7 +13,7 @@
     "url": "git+https://github.com/Brightspace/s-html.git"
   },
   "homepage": "https://github.com/Brightspace/s-html#readme",
-  "name": "s-html",
+  "name": "@brightspace-ui/s-html",
```

```suggestion
  "name": "@brightspace-ui-labs/s-html",
```
We've ben doing `@brightspace-ui-labs` for things in `BrightspaceUILabs`